package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "workshop-protractor",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Protractor",
+  "main": "index.js",
+  "scripts": {
+    "clean": "del /s dist",
+    "build": "npm run clean && tsc",
+    "test": "npm run build && protractor dist/protractor/local.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/manuelq12/protractor-workshop-2020.git"
+  },
+  "keywords": [
+    "ui-testing",
+    "dojo",
+    "practice",
+    "protractor"
+  ],
+  "author": "Manuel Quintero <quinteromanuel1202@gmail.com> (manuelq12)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/manuelq12/protractor-workshop-2020/issues"
+  },
+  "homepage": "https://github.com/manuelq12/protractor-workshop-2020#readme",
+  "dependencies": {
+    "protractor": "^7.0.0"
```

Comment:
this should be a dev dependency

Reply:
All of it?

Reply:
yes, there is an error on the steps, all dependencies on this section should be installed as dev dependencies